application.go

```diff
@@ -180,6 +181,13 @@ func (r *Application) CPU(cpu float64) *Application {
 	return r
 }
 
+// GPU set the amount of GPU per instance which is assigned to the application
+//		gpu:	the GPU (check MESOS docs) per instance
+func (r *Application) GPU(gpu float64) *Application {
```

Comment:
This change will break the build, returned message is `type Application has both field and method named GPUs`